projects/zetasql/Dockerfile

```diff
@@ -25,7 +25,7 @@ RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add - && \
 
 RUN apt-get update && apt-get -y install bazel-1.0.0
 
-RUN git clone -b arithmetic_fuzzer --depth 1 \
+RUN git clone -b master --depth 1 \
```

Comment:
-b master can be removed, not needed.

Reply:
So this one goes back to the question I asked in the Chatroom. I propose that we keep `-b` here because it's likely we'll point to branches for newly developed fuzzers in future OSS-Fuzz PRs often, if fuzzer branches don't get to merged to our `master` branch before we submit any OSS-Fuzz PR. Leaving `-b` here makes it so that we only need to change minimally the branch name. 

Let me know if you have better suggestions.